google/async_resumable_media/_download.py

```diff
@@ -357,10 +373,9 @@ async def _process_response(self, response):
             callback=self._make_invalid,
         )
         headers = self._get_headers(response)
-        response_body = self._get_body(response)
+        response_body = await self._get_body(response)
 
-        responsebody1 = copy.copy(response_body)
-        responsebody = await responsebody1.read()
+        response_body_content = response_body
```

Comment:
seems after a refactor this became a no-op? maybe collapse ln 376 and 378 to assign direct to `response_body_content`?

Reply:
Yep deleted this